tests/dummy/app/pods/error/controller.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember'
+const {Controller} = Ember
+
+export default Controller.extend({
+  // BEGIN-SNIPPET error-query-params
+  queryParams: {
+    showDetails: {
+      replace: true
+    }
+  },
+  showDetails: false,
```

Comment:
Shouldn't be external but can make this boolean an internal state variable based on our conversation this morning right?